### PR TITLE
Enable UFMT format on test/license.py test/logging.py 

### PR DESCRIPTION
**Pull request by zeshengzong**
Fixes some files in #123062

Run lintrunner on files:
test/license.py test/logging.py 

```bash
$ lintrunner -a --take UFMT --all-files
ok No lint issues.
Successfully applied all patches.
```

### CODE REVIEW

**Comment by ezyang** [comment]
There is a diff but whatever ig

```
diff --git b/test/test_license.py a/test/test_license.py
index e54247e9af3..bc6865ac47f 100644
--- b/test/test_license.py
+++ a/test/test_license.py
@@ -21,6 +21,7 @@ distinfo = glob.glob(os.path.join(site_packages, "torch-*dist-info"))
 
 
 class TestLicense(TestCase):
+
     @unittest.skipIf(not create_bundled, "can only be run in a source tree")
     def test_license_for_wheel(self):
         current = io.StringIO()
```

**Review by ezyang — APPROVED** [review]

**Review by malfet — APPROVED** [review]
LGTM, though you need to fix lint before landing (and if you've allowed approving suggested changes I could have fixed it for you)

**Comment by hippocookie** [comment]
> LGTM, though you need to fix lint before landing (and if you've allowed approving suggested changes I could have fixed it for you)

@malfet Not get the point from workflow error message, it's great if I can get some help, thanks!

File: test/test_license.py [hunk 1]
```diff
@@ -6,32 +6,34 @@
...
-license_file = 'third_party/LICENSES_BUNDLED.txt'
-starting_txt = 'The Pytorch repository and source distributions bundle'
+license_file = "third_party/LICENSES_BUNDLED.txt"
+starting_txt = "The Pytorch repository and source distributions bundle"
 site_packages = os.path.dirname(os.path.dirname(torch.__file__))
-distinfo = glob.glob(os.path.join(site_packages, 'torch-*dist-info'))
+distinfo = glob.glob(os.path.join(site_packages, "torch-*dist-info"))
+
 
 class TestLicense(TestCase):
```
Comment (malfet) [inline test/test_license.py:24]:
```suggestion
```

**Comment by malfet** [comment]
> Not get the point from workflow error message, it's great if I can get some help, thanks!

`ufmt` linter requirements has not been met, you have an extra empty line 23 of test_license.py

**Comment by hippocookie** [comment]
> > Not get the point from workflow error message, it's great if I can get some help, thanks!
> 
> `ufmt` linter requirements has not been met, you have an extra empty line 23 of test_license.py

@malfet Thanks for your help! 

I was running the command local env, doesn't seem to find that illegal extra empty line, is there might be some difference between local and workflow UFMT check?

Running command like: 
```bash
$ lintrunner -a --tabke UFMT <FileName>
# or
$ lintrunner -a --take UFMT --all-files
```

**Comment by malfet** [comment]
> I was running the command local env, doesn't seem to find that illegal extra empty line, is there might be some difference between local and workflow UFMT check?

There shouldn't be, UFMT is a pretty straightforward and stable linter. But do you have the right version? Do `lint runner init --take UFMT` and see if it changes the version you are running

**Comment by hippocookie** [comment]
> > I was running the command local env, doesn't seem to find that illegal extra empty line, is there might be some difference between local and workflow UFMT check?
> 
> There shouldn't be, UFMT is a pretty straightforward and stable linter. But do you have the right version? Do `lint runner init --take UFMT` and see if it changes the version you are running

Seems I have different version of "black", fixed after I re-install ufmt and black, Thanks for help:D @malfet

**Pull request closed without merging**